### PR TITLE
ISSD-4058 Article General Styling

### PR DESCRIPTION
**Pull request by rtienpal**

### CODE REVIEW

**Comment by rtienpal** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -302,39 +304,9 @@ pre[class*='language-'] {
...
-				font-weight: 600;
-				line-height: 1.5rem;
-				padding: 0 0 0 10px !important;
-
-				&:hover {
-					color: var(--color-neutral-10, #282934) !important;
-				}
-			}
-		}
+	.learn-article-wrapper {
```
Comment (ryanschuhler) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:307]: @rtienpal I dont think you did this, but can you make sure the .info-bar is correctly alphabetized? it looks like its not in the correct order

Comment (rtienpal) [reply workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:315]: I just sent new commit with the adjustment, although I searched throughout Learn's site and didn't find this element, maybe we should delete this css class?

**Comment by rtienpal** [comment]
ci:test:sf

**Comment by ryanschuhler** [comment]
thanks! https://github.com/brianchandotcom/liferay-portal/pull/149274

**Pull request closed without merging**